examples_tests/49.NewAPITest/main.cpp

```diff
@@ -33,394 +33,366 @@ static void debugCallback(video::E_DEBUG_MESSAGE_SEVERITY severity, video::E_DEB
 	std::cout << "OpenGL " << sev << ": " << msg << std::endl;
 }
 
```

Comment:
just filter out/revert all changes to this file (google "git cherrypick history")